src/utils/ui-utils.js

```diff
@@ -244,6 +244,13 @@ var UIUtils = function($) { // jshint ignore:line
             $iframeDocument.off('selectstart', cancelIFrameSelection);
         };
 
+        // prevent iframe dragging while browser tabs is switching
+        // modern browsers
+        document.addEventListener('visibilitychange', onMouseUp);
+        // old browsers
+        window.addEventListener('focus', onMouseUp);
+        window.addEventListener('blur', onMouseUp);
+
```

Comment:
I deleted focus and 'blur' 'event'. 'visibilitychange' is enough. See: http://caniuse.com/#feat=pagevisibility